### DISCUSSION TITLE
justify-between for flex inside grid

### DISCUSSION THREAD

**Discussion by Joyfred** [tailwindlabs/tailwindcss#6080] [Help] [answered]
I'm trying to equally space out items inside flex (with column as main axis) which is inside a grid. I'm not sure if I'm missing something. In the pen below, I expect 1,2,3,4 of first div to make use of its parent's space completely, with space between them equally split.

https://codepen.io/pen/?template=MWvPpjE
Thanks in advance 🙏

**Comment by NovusOpus** [comment, marked as answer]
You need to add: `flex` with `flex-col` as seen in [Tailwind CSS Docs examples](https://tailwindcss.com/docs/flex-direction), if you only add flex-col it's not going to work since you only defining the direction but not enabling flex itself.

**Reply by Joyfred** [reply]
Thanks a lot, @NovusOpus. Embarrassed for missing this😣
